src/Controller/ProcessOverviewController.php

```diff
@@ -39,6 +39,9 @@ public function show(ProcessOverviewGroup $group, ProcessOverview $overview): Re
                     'Go to page {page}' => t('Go to page {page}'),
                     'Go to next page' => t('Go to next page'),
                     'Missing data' => t('Missing data'),
+                    'Try again!' => t('Try again!'),
+                    'Copied!' => t('Copied!'),
+                    'Copy field' => t('Copy field'),
```

Comment:
I think we should be a little more informative here (even though it's more boring and corporate sounding):

```suggestion
                    'Try again!' => t('Error copying value. Try again!'),
                    'Copied!' => t('Value copied!'),
                    'Copy field' => t('Copy value'),
```